src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -146,15 +146,15 @@ export class SavedObjectsRepository {
    * @property {boolean} [options.overwrite=false]
    * @property {object} [options.migrationVersion=undefined]
    * @property {string} [options.namespace]
-   * @property {array} [options.references] - [{ name, type, id }]
+   * @property {array} [options.references=[]] - [{ name, type, id }]
```

Comment:
does this cause any change to the generated documentation ?

Reply:
we don't expose the SavedObjectsRepository from Core so no docs are generated for it.